src/components/MenuBars/MenuItem/MenuItem.scss

```diff
@@ -1,225 +1,229 @@
 @import "constants/style.scss";
 
+// DISABLED
+.menu-item:disabled {
+  cursor: not-allowed;
+}
+.menu-item:disabled .menu-item__icon {
+  color: #adadad;
```

Comment:
We use #a3a6aa as a disabled color in several components. Should it be the same?